sdk/identity/azure-identity/inc/ApiViewSettings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "sourceFilesToProcess": null,
```

Comment:
Should this include all the header files, such as:
https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/identity/azure-identity/inc/azure/identity.hpp

Reply:
If you don't specify a sourceFilesToProcess, the tool collects all the header files under the source root and imports them. See the examples in storage, identity, core, etc.

Reply:
Gotcha, that makes sense.

Some follow-up questions:
1) If you do specify a sourceFilesToProcess, but also list additional directories, will it collect and add a the header files from those additional directories?

2) Also, is sourceFilesToProcess is not null, will it not collect the headers under that source root? As in, it's one of the other, not both?

3) What would be the source root? Is it where the service library specific src/ folder is located?

Reply:
Additional directories is the "-I" switch to the compiler. For example, the Identity SDK uses `additionalIncludeDirectories` to reference Azure Core. Otherwise we would have missing type errors in identity.

The source root is whatever you specify as the input directory to the tool.
If you specify sourceFIlesToProcess, it means you have a specific set of files you want to be included in the API View. If you don't specify sourceFilesToProcess, it means you want every file to be included in the API view.

FWIW, this is exactly the feature you requested :).